renovate.json

```diff
@@ -10,7 +10,8 @@
   "enabledManagers": [
```

Comment:
No need to be explicit here, we can fully remove the `enabledManagers` and let renovate autodetect